libraries/resource_dot_my_cnf.rb

```diff
@@ -0,0 +1,21 @@
+class Chef
+  class Resource
+    # used to create and managed .my.cnf files
+    class MysqlmDotMyCnf < Chef::Resource::LWRPBase
+      resource_name :mysqlm_dot_my_cnf
```

Comment:
I think this can be omitted since the default algorithm for converting the class name to a resource name produces `mysqlm_dot_my_cnf` already.